src/components/Banner/Banner.stories.tsx

```diff
@@ -31,7 +31,7 @@ const getDescription = (status?: Variant) => (
     curriculum. To see the updates,{' '}
     <Button
       href="/"
-      onClick={(event) => event.preventDefault()}
+      onClick={(event: any) => event.preventDefault()}
```

Comment:
choosing to mostly ignore `any` types in stories

Reply:
Sounds good. For this case I'm wondering why the `onClick` isn't getting the typing from `ClickableStyle` (event: MouseEventHandler)

Reply:
Agreed, the type should definitely be coming from Button.

Looking at Button component (and its stories), I think we're actually getting `any` for all of it's props.

Not exactly sure why, but it looks like at least `ClickableStyleProps` is resulting in `any`. But something's going on there for sure 🤔

Reply:
ohhhh shoot excellent catch -- we were typing Button with `ClicakbleStyleProps<React.Element>` 😮